Distance_3/include/CGAL/squared_distance_3_2.h

```diff
@@ -286,7 +286,7 @@ squared_distance_to_triangle_RT(
       {
         // the projection of pt is inside the triangle
         inside = true;
-        return squared_distance_to_plane_RT(normal, vector(t0, pt), num, den, k);
+        squared_distance_to_plane_RT(normal, vector(t0, pt), num, den, k);
```

Comment:
Now a `return` is missing.
```suggestion
        squared_distance_to_plane_RT(normal, vector(t0, pt), num, den, k);
        return;
```

Reply:
Well, with the `if`/`else`, that return is not mandatory.